src/Properties/LibraryProperties.php

```diff
@@ -91,15 +93,25 @@ public static function new(string $composerJsonFile, ?string $baseUrl = null): L
     }
 
     /**
-     * @param string $packageName
+     * @param array $composerJsonData
+     * @param string $composerJsonFile
      *
-     * @return string
+     * @return array{string, string, string}
      */
-    private static function buildBaseName(string $packageName): string
+    private static function buildNames(array $composerJsonData, string $composerJsonFile): array
```

Comment:
the method name is might be a bit wrong when we return the `$basePath` as well (which is not a name). 

I'm wondering why the `$basePath` was moved into this method when it is just returned and not used for anything else. The `$composerJsonFile` as param is only used for that as far as i can see. Would it make sense to move that out again to keep the method smaller?

Reply:
At first I wanted to have a fallback for basename, in the case the `name` property was not defined in Composer data.

It looked like:

```php
$composerName = $composerJsonData['name'] ?? dirname($composerJsonFile);
```

So I needed the `composer.json` path. But then I decided to remove that part.

Reply:
Fixed 2455d05096b29b4b710d02f7a9f0dcac959b4c2a